tests/logout.test.js

```diff
@@ -0,0 +1,35 @@
+import LoginPage from '../pages/login.page';
+import LogoutPage from '../pages/logout.page';
+import HomePage from '../pages/home.page';
+import BasePage from '../pages/base.page';
+
+const LOGIN_PAGE = LoginPage;
```

Comment:
Same comment as above, this is not needed as LoginPage is already an instantiated object, you can directly use it after importing it